docs/frameworks.md

```diff
@@ -49,3 +52,13 @@ layout: documentation
 ## Machine Learning
 
 * [TFMesos](https://github.com/douban/tfmesos) is a lightweight framework to help running distributed [Tensorflow](https://www.tensorflow.org/) Machine Learning tasks on Apache Mesos with GPU support.
+
+## Load Balancing
+
+* [Traefik Mesos provider](https://github.com/AVENTER-UG/traefik-mesos) is a modern HTTP reverse proxy and load balancer with build in TCP and UDP support.
+
+## Executor
+
```

Comment:
I do not think we should put it here since this doc is `frameworks.md`.

Reply:
:thinking: ok, maybe we should have a place where we can add community executors? :man_shrugging: